torch/csrc/jit/pickler.cpp

```diff
@@ -592,11 +592,11 @@ OpCode Unpickler::readInstruction() {
           AT_ERROR("Unknown list specialization");
         }
       } else {
-        stack_.emplace_back(std::vector<IValue>());
+        stack_.emplace_back(c10::impl::make_generic_list());
```

Comment:
I am not sure why we are regressing the API. Why can't we convert a std::vector into a list?